elements/pfe-navigation/src/pfe-navigation-item.scss

```diff
@@ -0,0 +1,189 @@
+    --pfe-broadcasted--color--ui-link--hover:   #{pfe-local(Color)};
+    --pfe-broadcasted--color--ui-link--visited: #{pfe-local(Color)};
+    --pfe-broadcasted--color--ui-link--focus:   #{pfe-local(Color)};
+
+    @each $level in (alpha, beta, gamma) {
+      --pfe-theme--font-size--#{$level}: #{pfe-local($cssvar: FontSize, $region: trigger)};
+      --pfe-theme--font-weight--#{$level}: #{pfe-local($cssvar: FontWeight, $region: trigger)};
+    }
+
+    transition: all #{pfe-var(animation-timing)};
+
+    position: relative;
+
+    display: flex;
+    flex-flow: column nowrap;
+    align-items: center;
+    justify-content: center;
+    flex-grow: 1;
+  
+    font-family: #{pfe-var(font-family)};
+    text-align: center;
+    font-size: #{pfe-local($cssvar: FontSize, $region: trigger)};
+    font-weight: #{pfe-local($cssvar: FontWeight, $region: trigger)};
+    color: #{pfe-local(Color)};
+  
+    margin: 0;
+    white-space: nowrap;
+    outline: none;
+
+    background-color: #{pfe-local(BackgroundColor)};
+    padding: var(--pfe-navigation--Padding--vertical) var(--pfe-navigation-item--Padding--horizontal) !important;
+
+    border-top: #{pfe-var(surface--border-width--heavy)} #{pfe-var(surface--border-style)} #{pfe-local(BorderTopColor)};
+    border-right: #{pfe-var(surface--border-width)} dashed #{pfe-local(BorderColor)};
+    border-bottom: #{pfe-var(surface--border-width)} dashed #{pfe-local(BorderColor)};
+    border-left: #{pfe-var(surface--border-width)} dashed #{pfe-local(BorderColor)};
+    color: #{pfe-local(Color)};
+
+    &:hover,
+    :host([aria-current="location"]) {
+      --pfe-navigation-item--BorderTopColor:  #{pfe-color(surface--accent)};
+    }
+
+    &:focus {
+      --pfe-navigation-item--BorderTopColor:  #{pfe-color(surface--accent)};
+      --pfe-navigation-item--BorderColor: #{pfe-color(surface--lightest)};
+    }
+
+    &[aria-expanded="true"],
+    .expanded & {
+      --pfe-navigation-item--Color: #{pfe-color(text)};
+      --pfe-navigation-item--BackgroundColor: #{pfe-color(surface--lightest)};
+      --pfe-navigation-item--BorderTopColor:  #{pfe-color(surface--accent)};
+    }
+
+    &:focus:not(:focus-visible) {
+      outline: none;  // removes outline for mouse users, preserves it for keyboard users
+    }
+  
+    :host([pfe-icon]) & {
+      --pfe-navigation-item__trigger--FontSize: 12px;
+    }
+
+    &--icon {
+      display: flex;
+      background: #{pfe-local($cssvar: Background, $region: icon)};
+
+      :host(:not([pfe-icon])) [aria-expanded="false"] & {
+        visibility: hidden;
+      }
+
+      :host(:not([pfe-icon])) & {
+        @include pfe-arrow(down, #{pfe-color(feedback--default)}, $width: 6px, $height: 6px) {
+          position: absolute;
+          bottom: 13px;
+        }
+      }
+
+      :host([pfe-icon]) & {
+        margin-bottom: calc(#{pfe-var(container-spacer)} / 2);
+        width: 21px;
+        height: 21px;
```

Comment:
Should we set a local variable for these?

Reply:
Let's leave them hardcoded for now and we can revisit hooks later.